app/src/main/java/com/lfcounago/gastoscompartidos/GroupProfileActivity.java

```diff
@@ -0,0 +1,408 @@
+                                                                public void onSuccess(Void unused) {
+                                                                    // Éxito al actualizar el nombre del grupo en la colección "spends"
+                                                                    Toast.makeText(getApplicationContext(), "Nombre del grupo actualizado en la colección 'spends'", Toast.LENGTH_SHORT).show();
+                                                                    // Actualizar la interfaz de usuario
+                                                                    etName.setText(nuevoNombre);
+                                                                    getGroupData();
+                                                                    toGroupDetails();
+                                                                    // Refrescar el listview también por si acaso
+                                                                    adapter.notifyDataSetChanged();
+                                                                }
+                                                            })
+                                                            .addOnFailureListener(new OnFailureListener() {
+                                                                @Override
+                                                                public void onFailure(@NonNull Exception e) {
+                                                                    // Error al intentar actualizar el nombre del grupo en la colección "spends"
+                                                                    Toast.makeText(getApplicationContext(), "No se pudo actualizar el nombre del grupo en la colección 'spends'", Toast.LENGTH_SHORT).show();
+                                                                }
+                                                            });
+                                                }
+                                            }
+                                        }
+                                    }
+                                });
+                    }
+                })
+                .addOnFailureListener(new OnFailureListener() {
+                    @Override
+                    public void onFailure(@NonNull Exception e) {
+                        // Error al intentar actualizar el nombre del grupo en la colección "groups"
+                        Toast.makeText(getApplicationContext(), "No se pudo actualizar el nombre del grupo", Toast.LENGTH_SHORT).show();
+                    }
+                });
+    }
+
+
+    // Método para cuando se edita un grupo volver a la pantalla principal de los gastos del grupo GroupDetailsActivity
+    private void toGroupDetails() {
+        // Crear un Intent para iniciar la actividad de detalles del grupo
+        Intent intent = new Intent(this, ListUserGroupsActivity.class);
```

Comment:
Después de guardar el grupo debe devolver al usuario al escenario anterior 3, no a la lista de grupos del usuario

Reply:
Solve with  [fix: Solve redirection problems in toGroupDetails function](https://github.com/lfcounago/GastosCompartidos/pull/43/commits/0e48b1c0a5dfa5972c8903cb5dea3dc73fd1d949)